emulators/tnylpo/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           gitlab 1.0
+
+name                tnylpo
+gitlab.setup        gbrein tnylpo 1.1.3
+revision            0
+
+categories          emulators
+license             BSD
+maintainers         {@aeiouaeiouaeiouaeiouaeiouaeiou outlook.com:aeioudev} openmaintainer
+
+description         Utility to run CP/M-80 programs under Unix/Linux
+long_description    ${name} allows the execution of programs written \
+                    for CP/M-80 version 2.2 under Unixy operating systems.
+
+checksums           rmd160  3c2d754659e99da638edfadd0b52921dca581e3b \
+                    sha256  f017db288d4de2aceeacc7a507e789bdd399cfc23a818c556117ff87c2fa3fe2 \
+                    size    83457
+
+depends_build       port:gmake
+depends_lib         port:ncurses
+
+use_configure       no
```

Comment:
you do set the compiler below, but are the other flags that MacPorts sets also propagated (e.g., CFLAG, LDFLAGS, ....). I do see for example an `-O3` flag in the Makefile, that is not what MacPorts uses. Can you please double-check this?